web/src/hooks/useWaku.ts

```
@@ -241,65 +324,148 @@ export function useWaku(): UseWakuReturn {
+            }
+          }, 2000);
+
+          return () => {
+            console.log(
+              "🧹 Cleaning up ENHANCED Waku handlers for address:",
+              address
+            );
             unsubscribeTestimonials();
             unsubscribeNotifications();
           };
         } catch (error) {
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)